templates/editEvents.html

```diff
@@ -2,9 +2,16 @@
   <div class="container-fluid">
 
     <div id="editEventsList">
-      <h2>
-        Events
-      </h2>
+      <div class="row">
+        <div class="col-xs-6">
+          <h2>
+            Events
+          </h2>
+        </div>
+        <div class="col-xs-6 text-right">
+            <span id="toggleCollapseAllEvents">Expand/Collapse All</span>
```

Comment:
nitpick: 4 spaces here

Oddly enough, these 2 don't really look like they're on the same row:
![2015-01-24_971x284_yogart](https://cloud.githubusercontent.com/assets/277474/5886756/e36417ae-a363-11e4-9395-3129f6d0c781.png)

I'm not sure why this is, or if this intentional.

Hmm... how do you feel about making this a button instead of something that looks like a link? I tried pretty hard to make it so that links that you to 2 pages, and buttons affect the current page, but don't change the url.